test/ch/aggregation_test.exs

```diff
@@ -19,10 +16,11 @@ defmodule Ch.AggregationTest do
     ORDER BY (ticker, time)
     """
 
-    Ch.query!(conn, create_table, %{})
+    Ch.query!(conn, create_table)
+    on_exit(fn -> Ch.Test.sql_exec("drop table ch_candle_fragments") end)
```

Comment:
I change the order of creating and dropping table to ensure we fail if some other contributor already has `candle_fragments` table. And I also prefix it with `ch_`. The proper fix is to create a database for `ch` tests like in `chto`, I'll do it tomorrow.